src/Logging/messages.lua

```diff
@@ -26,6 +28,7 @@ return {
 	forValuesProcessorError = "ForValues callback error: ERROR_MESSAGE",
 	forValuesDestructorError = "ForValues destructor error: ERROR_MESSAGE",
 	invalidChangeHandler = "The change handler for the '%s' property must be a function.",
+	invalidAttributeChangeHandler = "The attribute change handler for the '%s' property must be a function.",
```

Comment:
Is this a necessary distinction to make in the message?
If it is, the current way of specifying "attribute change handler for the [] property" is confusing. I would recommend putting "the change handler for the [] attribute"

Reply:
Yeah that sounds clearer to me, will update!